web/src/.env

```diff
@@ -0,0 +1 @@
+CURRENT_USER='A'
```

Comment:
This environment variable sets the default user at app initialization.

Reply:
So this is just effectively a shim for the POC correct? This isn't how we would do this when the app was functionally deployed?

Reply:
Yep, exactly. In the real app, we will want auth and sign in page. This and the dropdown to select the current user are just a convenience for the POC.